### PR TITLE
ColdWasabi: update using hww step-by-step

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

File: docs/using-wasabi/ColdWasabi.md [hunk 1]
```diff
@@ -13,16 +13,17 @@
 
 ## Using hardware wallet step-by-step
 
-1. Start your Wasabi Wallet and connect your hardware wallet with USB.
-Alternatively, you can import a Coldcard skeleton file via SD card.
-2. The `Hardware Wallet` tab will open, and there you can search all connected hardware wallets.
-3. Click `Load Wallet`, then you can [receive](/using-wasabi/Receive.md) bitcoin to addresses controlled by the hardware wallet.
-4. You can [spend](/using-wasabi/Send.md) these coins in the `Send` tab, though the hardware wallet must be connected via USB to confirm before signing the transaction.
+1. Start your Wasabi Wallet and go to `Add Wallet`.
+2. Click on `Connect to hardware wallet` and then connect the hardware wallet via cable. Alternatively, you can import a Coldcard skeleton file via SD card by using `Import Wallet` at the add wallet dialog.
```
Comment (yahiheb) [inline docs/using-wasabi/ColdWasabi.md:17]:
```suggestion
2. Click on `Connect to hardware wallet` and then connect the hardware wallet via cable. Alternatively, you can import a Coldcard skeleton file via SD card by using `Import Wallet` at the `Add Wallet` dialog.
```

File: docs/using-wasabi/ColdWasabi.md [hunk 2]
```diff
@@ -13,16 +13,17 @@
...
+3. Confirm that the detected hardware wallet is correct.
+4. Click `Open`, and wait for the wallet to load.
+5. Now you can [receive](/using-wasabi/Receive.md) bitcoin to addresses controlled by the hardware wallet.
+6. You can [spend](/using-wasabi/Send.md) these coins in the `Send` dialog, though the hardware wallet must be connected via USB to confirm before signing the transaction.
```
Comment (yahiheb) [inline docs/using-wasabi/ColdWasabi.md:21]:
```suggestion
6. You can [spend](/using-wasabi/Send.md) these coins in the `Send` dialog, though the hardware wallet must be connected via USB to sign the transaction.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Comment by yahiheb** [comment]
How about deleting all of this, it is redundant and #1317 will explain the steps in details.

**Comment by MarnixCroes** [comment]
> How about deleting all of this, it is redundant and #1317 will explain the steps in details.

imo it's ok to leave this for now
.
This functions as short step overview, where later on each step is collaborated on
note that, most other _using wasabi_ also have these step by step

**Review by yahiheb — APPROVED** [review]

**Pull request merged**